back/Gemfile.lock

```diff
@@ -70,12 +70,6 @@ PATH
       rails (~> 7.0)
       ros-apartment (>= 2.9.0)
 
-PATH
-  remote: engines/commercial/geographic_dashboard
```

Comment:
Weird that we don't seem to have a feature flag for this, probably already removed before?

Reply:
Yes we did, it's implemented through the feature specification of the engine. In this PR, the flag will be removed.

Reply:
Ah yes, of course :+1: